ci/tests/system/validation_after_test.py

```diff
@@ -132,9 +132,15 @@ def test_basic_logrotate():
                         msg='Logrotate on Storage Router {0} does not include {1}'.format(storagerouter.name, logrotate_include_dir))
             assert_true(expr='/usr/sbin/logrotate /etc/logrotate.conf' in root_client.file_read(filename=logrotate_cron_file).splitlines(),
                         msg='Logrotate will not be executed on Storage Router {0}'.format(storagerouter.name))
-            assert_equal(first=expected_logrotate_content,
-                         second=root_client.file_read(filename=logrotate_ovs_file),
-                         msg='Logrotate contents does not match expected contents on Storage Router {0}'.format(storagerouter.name))
+            actual_file_contents = root_client.file_read(filename=logrotate_ovs_file)
+            if actual_file_contents.endswith('\n'):
+                assert_equal(first=expected_logrotate_content,
+                             second=actual_file_contents,
+                             msg='Logrotate contents does not match expected contents on Storage Router {0}'.format(storagerouter.name))
+            else:
```

Comment:
Why strip off the new line?  I'd expect the logrotate file to always have the same contents
We should investigate why there is a difference rather than working around this